File: nhglib-core/src/io/github/movementspeed/nhglib/input/interfaces/InputListener.java
package io.github.movementspeed.nhglib.input.interfaces;

import io.github.movementspeed.nhglib.input.models.NhgInput;

/**
 * Created by Fausto Napoli on 09/01/2017.
 */
public interface InputListener {
    void onKeyInput(NhgInput input);

    void onStickInput(NhgInput input);

    void onPointerInput(NhgInput input);

    void onMouseInput(NhgInput input);
}

File: nhglib-core/src/io/github/movementspeed/nhglib/input/interfaces/InputListener.kt
package io.github.movementspeed.nhglib.input.interfaces

import io.github.movementspeed.nhglib.input.models.base.NhgInput

/**
 * Created by Fausto Napoli on 09/01/2017.
 */
interface InputListener {
    fun onInput(input: NhgInput)
}
